### ISSUE TITLE
Proposal: .knoxignore file for excludes

### ISSUE THREAD

**Issue by ma-hartma** [knoxite/knoxite#167]
I'd find the possibility of a .knoxignore/.knoxiteignore file to define exludes pretty nice.

**Comment by dominikheinz** [comment]
I agree, it would be really to have such functionality. It would allow one to do fine-grained filtering which files should be/shouldn't be backed up.
